### ISSUE TITLE
historical/edge cases. 

### ISSUE THREAD

**Issue by jph5396** [MDShields7/sumo-rank#19]
Hi @MDShields7,
I'm thinking of using sumo-rank in my projects, but I'm going to run into a few edge cases that might need to be handled. I figured I'd open an issue and see what you think about them. 

Limits: 
Im working with data as far back as Jan 1958 and the number of wrestlers in a division has changed a few times since then. the [Jan 1958 banzuke](http://sumodb.sumogames.de/Banzuke.aspx?b=195801&heya=-1&shusshin=-1) has wrestlers ranked at M20 and J24. I think the easiest way to handle this issue would be to add  an optional parameter in the format function that can be used to ignore limits. 

edge case ranks: 
Have you thought about adding format support for the additional designations that can be appended to the end of ranks (HD, TD, and YO)? I think it would be beneficial for the format function to at least allow them in the parameter, even if it just ignores them. HD is pretty common in data pre 1995, and TD is still used today. 

Let me know what you think about these issues. If you are wiling to accept pull requests for this, I'd be more than happy to contribute!

**Comment by MDShields7** [comment]
Yes, I think a 3rd parameter can be added, no problem. I will accept a pull request, and any other advice as no one else has really looked at the guts of this library.

Just wondering, are you using this in the browser, or NodeJS? I have been rebuilding this in a big way with testing for NodeJS (not pushed yet). About at 90% code coverage now, I believe.

Also, just wondering what kind of project? Any other sumo-related library I can work on?

**Comment by jph5396** [comment]
great! I'll take a crack at it this week. I'm using it in a browser.

I've been slowly putting together a  [website](https://www.sumostatsitics.org) for elo ratings I've calculated as a personal/learning project. The site is in a pretty simple state right now but I figured anything is useful when there are so few good sumo resources in english.
